src/components/FooterLinks.astro

```diff
@@ -0,0 +1,51 @@
+---
+import { Icon } from '@astrojs/starlight/components';
+import FeedbackButton from '../components/tutorial/FeedbackButton.astro';
+---
+
+<section class="sl-flex" style="justify-content: center; flex-wrap: wrap;">
+	<a
+		href="https://docs.astro.build/en/contribute/"
+		class="sl-flex"
+		style="display: flex; flex-direction: row; align-items: center; text-decoration: none;"
+	>
+		<Icon name="open-book" size="1.2em" />
+		 Contribute
```

Comment:
Question: Should this not be translated?

Reply:
I can review this. Perhaps I missed the translation for this.